### DISCUSSION TITLE
User Relationship Assigned to Created Item

### DISCUSSION THREAD

**Discussion by perezstudio** [Budibase/budibase#11340] [Help] [answered]
In my database I've got 3 tables all of which have a relationship tying them together.
1. Videos
2. Campaigns
3. Users
Users is a many to many relationship with Campaigns and Campaigns has a 1 to many relationship with Videos
Currently a user is assigned to a campaign. When a user creates a Video Item has only 2 fields but neither sets the campaign the video is related to. That is pulled from the campaign the user is assigned to.

I've attempted making relationships between the video item and the campaign using the Users.Campaign._id assigned to the Videos.Campaigns field but no luck. Would love some guidance on how to accomplish this.

Thanks,

**Comment by melohagan** [comment, marked as answer]
Hey @perezstudio 

This binding `Users.Campaign._id` won't work. This is because relationships are always stored in an array (even if there is only one relationship).

You could try instead `Users.Campaign.0._id` which would get the _id of the first campaign relationship, although I don't recommend this because you can't guarantee the order of the relationships if there's more than one.

These may be useful:

 - https://docs.budibase.com/docs/form-steps
 - https://docs.budibase.com/docs/chaining-actions#example-creating-and-linking-two-table-entries-in-one-form

**Reply by perezstudio** [reply]
Big thanks for the help! I did have to put the 0 in an array as follows "{{ Current User.Campaign.[0]._id }}" and it worked. While you were right that this won't work for when there are multiple "campaigns" assigned, my use case doesn't have this problem. Just want to document that for others that might find this solution.
